api/MediaStreamTrack.json

```diff
@@ -172,10 +172,10 @@
               "version_added": "44"
             },
             "safari": {
-              "version_added": false
+              "version_added": "12.1"
```

Comment:
This is `contentHint`, makes sense as introduced after `MediaStreamTrack` itself. I know it was in Chrome at least.